src/demand.h

```diff
@@ -3,61 +3,45 @@
  Project:      OWA EPANET
  Version:      2.2
  Module:       demand.h
- Description:  data for demand pattern list
+ Description:  manages list of demand categories
  Authors:      see AUTHORS
  Copyright:    see AUTHORS
  License:      see LICENSE
- Last Updated: 04/12/2019
+ Last Updated: 04/26/2019
  ******************************************************************************
 */
 
 #ifndef DEMAND_H
 #define DEMAND_H
 
+typedef struct list_s list_t;
 
-#include "util/list.h"
+typedef struct demand_data_s
```

Comment:
None of the other data structs defined in types.h (for nodes, links, patterns, etc.) obeys encapsulation so why start now. We hide list.h because it's only needed in demand.c, and we hide its list_node_s struct because that is an implementation detail -- if it changes we shouldn't have to change the interface to a list.